.github/workflows/on_pr_master.yml

```diff
@@ -45,9 +67,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   build:
+    needs: changes
     # Macos includes virtualbox and vagrant.
     runs-on: macos-latest
-
+    if: ( ( github.event.action == 'closed' && github.event.pull_request.merged == true ) && contains(github.event.pull_request.labels.*.name, 'major')||contains(github.event.pull_request.labels.*.name, 'minor')||contains(github.event.pull_request.labels.*.name, 'patch') ) || ( github.event.action != 'closed' && needs.changes.outputs.not-doc == 'true' )
```

Comment:
Can we split it and arrange this piece of cod in multi lines to improve readability?

Reply:
Yes.